Content.Server/GameObjects/Components/Utensil/UtensilComponent.cs

```diff
@@ -20,19 +22,19 @@ namespace Content.Server.GameObjects.Components.Utensil
     [RegisterComponent]
     public class UtensilComponent : SharedUtensilComponent, IAfterInteract
     {
-        [Dependency] private readonly IEntitySystemManager _entitySystem = default!;
-        [Dependency] private readonly IRobustRandom _random = default!;
-
-        protected UtensilType _types = UtensilType.None;
+        private UtensilType _types = UtensilType.None;
 
         [ViewVariables]
         public override UtensilType Types
         {
             get => _types;
             set
             {
+                if (_types.Equals(value))
+                    return;
+
                 _types = value;
-                Dirty();
+                //Dirty();
```

Comment:
You can remove the SharedUtensilComponent class entirely if you are going to remove the dirty call, given there's no utensil component on the client.

Reply:
I assumed you'd predict it someday, though given tag component's coming you could probably replace this component right.